Userland/Services/FileSystemAccessServer/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+serenity_component(
+    FileSystemAccessServer
+    REQUIRED
+    TARGETS FileSystemAccessServer
+)
+
+compile_ipc(FileSystemAccessServer.ipc FileSystemAccessServerEndpoint.h)
+compile_ipc(FileSystemAccessClient.ipc FileSystemAccessClientEndpoint.h)
+
+set(SOURCES
+    ClientConnection.cpp
+    DownloadWidget.cpp
+    main.cpp
+    FileSystemAccessServerEndpoint.h
+    FileSystemAccessClientEndpoint.h
+)
+
+serenity_bin(FileSystemAccessServer)
+target_link_libraries(FileSystemAccessServer LibCore LibIPC LibGUI LibWeb LibProtocol LibDesktop)
```

Comment:
You should probably use the new `serenity_component` directive here. Also, the programs which now depend on the new Service should declare it as dependency in their `serenity_component` declaration.